k9mail/src/test/java/com/fsck/k9/message/html/HtmlConverterTest.java

```diff
@@ -207,4 +207,110 @@ public void testLinkifyBitcoinAndHttpUri() {
                 "http://example.com/" +
                 "</a>", outputBuffer.toString());
     }
+
+    @Test
+    public void testIssue2259Spec() {
```

Comment:
Please drop the `test` prefix from all test methods. The `@Test` annotation already marks the methods as tests.